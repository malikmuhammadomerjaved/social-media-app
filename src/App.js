import { lazy, Suspense } from "react";
import { BrowserRouter, Routes, Route } from "react-router-dom";

import "./App.scss";

const Home = lazy(() => import("./pages/home"));
const Events = lazy(() => import("./pages/events"));

function App() {
  return (
    <BrowserRouter>
      <Suspense fallback={<div>Loading...</div>}>
        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/event" element={<Events />} />
        </Routes>
      </Suspense>
    </BrowserRouter>
  );
}

export default App;
