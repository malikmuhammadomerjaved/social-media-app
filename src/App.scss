@import "./styles/commons.scss";

html {
  font-size: 10px;

  * {
    padding: 0;
    margin: 0;
    box-sizing: border-box;
    font-family: "Helvetica";
  }

  body {
    background: linear-gradient(
        0deg,
        $primary-background-1,
        $primary-background-1
      ),
      $primary-background-2;

    h1 {
      font-size: 3.6rem;

      @media (min-width: $mobile-screen) {
        font-size: 6.4rem;
        line-height: 6.4rem;
      }
    }

    h2 {
      font-size: 1.6rem;

      @media (min-width: $mobile-screen) {
        font-size: 2.4rem;
        line-height: 1.8rem;
      }
    }

    h3 {
      font-size: 2rem;
    }
  }
}
