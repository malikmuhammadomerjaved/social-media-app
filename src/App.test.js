import { render } from "@testing-library/react";
import App from "./App";

describe("<App />", () => {
  const { container } = render(<App />);

  it("should match App component to snapshot", () => {
    expect(container).toMatchSnapshot();
  });
});
