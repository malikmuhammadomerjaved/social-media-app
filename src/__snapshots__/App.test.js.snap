// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match App component to snapshot 1`] = `
<div>
  <div>
    Loading...
  </div>
</div>
`;
